packages/core/src/components/context-menu/contextMenuSingleton.tsx

```diff
@@ -50,7 +50,9 @@ export function showContextMenu(
 ) {
     const {
         container = document.body,
+        // eslint-disable-next-line deprecation/deprecation
```

Comment:
Can we add some sort of react 18 note to these comments so that we can easily track these down later?

Reply:
Yep! Added here eb3f5e367b647d93dedf4ae448c11d88bf6b1167